src/game/components/Model.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef GAME_COMPONENTS_MODEL_HPP_
+#define GAME_COMPONENTS_MODEL_HPP_
+
+#include <filesystem>
+#include "ecs/Component.hpp"
+#include "raylib/model/Model.hpp"
+// #include "raylib/model/Mesh.hpp"
+
+namespace game
+{
+    struct Model : public ecs::Component, public raylib::model::Model {
+        Model(raylib::model::Model &pmodel) : raylib::model::Model(pmodel) {}
+        // Model(const raylib::model::Mesh &pmesh) : raylib::model::Model(pmesh) {}
```

Comment:
Why is this constructor commented ?

Reply:
because I don't have the Model class in this branch yet

Reply:
need #105